src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -123,7 +131,7 @@ const RadioGroup = React.createClass(createLucidComponentDefinition({
 								callbackId={index}
 								name={name}
 								onSelect={this.handleSelected}
-								Label={_.get(_.first(RadioGroup.Label.findInAllAsProps(radioButtonChildProp)), 'children', '')}
+								Label={_.get(_.first(findTypes(radioButtonChildProp, RadioGroup.Label)), 'props.children', null)}
```

Comment:
this was a weird bug that broke enzyme. Changing `''` to `null` fixed it, but it shouldn't be breaking anyone's code.